projects/prometheus/prometheus/README.md

```diff
@@ -0,0 +1,15 @@
+## **Prometheus**
+
```

Comment:
I *thought* these readmes were supposed to have a version badge and build status, for example:

```
![Version](https://img.shields.io/badge/version-v0.1.1-blue)
![Build Status](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiN09rZ2FUVHBEMUZXeEV6VlBDLzEyNEVsN0FrU1d4U1pUUDQ5KzFlNGtyQjFVMitKRkRpSENRZzEyUUl3bjQwMUl3NElTQjc3T2krNFRaZGwvYWpKR0JvPSIsIml2UGFyYW1ldGVyU3BlYyI6InBvQ0lRWmVMODR4UG45NGMiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
```

Reply:
Added